pennylane/templates/layers.py

```diff
@@ -16,19 +16,20 @@
 They implement a transformation from a quantum state to another quantum state.
 """
 #pylint: disable-msg=too-many-branches,too-many-arguments,protected-access
+from pennylane import numpy as np
 from pennylane.ops import CNOT, RX, RY, RZ, Rot, Squeezing, Displacement, Kerr
 from pennylane.templates.subroutines import Interferometer
-import numpy as np
+from pennylane.templates.utils import (_check_shape, _check_no_variable, _check_wires,
+                                       _check_type, _check_number_of_layers, _check_shapes,
+                                       _get_shape)
 
 
-def _strongly_entangling_layer(weights, wires, r=1, imprimitive=CNOT):
+def _strongly_entangling_layer(weights, wires, r, imprimitive):
```

Comment:
I made all arguments positional, since when used internally they should all be specified!